doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCertificateStoreProvider.xml

```diff
@@ -1,61 +1,84 @@
+      </summary>
+    </SqlColumnEncryptionCertificateStoreProvider>
+    <ctor>
+      <summary>
+        Key store provider for Windows Certificate Store.
+      </summary>
+    </ctor>
+    <DecryptColumnEncryptionKey>
+      <param name="masterKeyPath">
+        The master key path.
+      </param>
+      <param name="encryptionAlgorithm">
+        The encryption algorithm. Currently, the only valid value is: <c>RSA_OAEP</c>
+      </param>
+      <param name="encryptedColumnEncryptionKey">
+        The encrypted column encryption key.
+      </param>
+      <summary>
+        Decrypts the specified encrypted value of a column encryption key. The encrypted value is expected to be encrypted using the certificate with the specified key path and using the specified algorithm. The format of the key path should be "Local Machine/My/&lt;certificate_thumbprint&gt;" or "Current User/My/&lt;certificate_thumbprint&gt;".
+      </summary>
+      <returns>
+        Returns <see cref="T:System.Byte" />. The decrypted column encryption key.
+      </returns>
+    </DecryptColumnEncryptionKey>
+    <EncryptColumnEncryptionKey>
+      <param name="masterKeyPath">
+        The master key path.
+      </param>
+      <param name="encryptionAlgorithm">
+        The encryption algorithm. Currently, the only valid value is: <c>RSA_OAEP</c>
+      </param>
+      <param name="columnEncryptionKey">
+        The plaintext column encryption key.
+      </param>
+      <summary>
+        Encrypts a column encryption key using the certificate with the specified key path and using the specified algorithm. The format of the key path should be "Local Machine/My/&lt;certificate_thumbprint&gt;" or "Current User/My/&lt;certificate_thumbprint&gt;".
+      </summary>
+      <returns>
+        Returns <see cref="T:System.Byte" />. The encrypted column encryption key.
+      </returns>
+    </EncryptColumnEncryptionKey>
+    <ProviderName>
+      <summary>
+        The provider name.
+      </summary>
+    </ProviderName>
+    <SignColumnMasterKeyMetadata>
+      <param name="masterKeyPath">
+        The column master key path.
+      </param>
+      <param name="allowEnclaveComputations">
+        <see langword="true" /> to indicate that the column master key supports enclave computations; otherwise, <see langword="false" />.
+      </param>
+      <summary>
+        Digitally signs the column master key metadata with the column master key referenced by the <paramref name="masterKeyPath" /> parameter.
+      </summary>
+      <returns>
+        The signature of the column master key metadata.
+      </returns>
+    </SignColumnMasterKeyMetadata>
+    <VerifyColumnMasterKeyMetadata>
+      <param name="masterKeyPath">
+        The complete path of an asymmetric key. The path format is specific to a key store provider.
+      </param>
+      <param name="allowEnclaveComputations">
+        A Boolean that indicates if this key can be sent to the trusted enclave.
```

Comment:
'Boolean' can be a class cref.

Reply:
It can ... but does it need to be? From my point of view, bool is a pretty obvious type to a C# developer, and if I have to go through and add links for all of these, it's really going to drag out how long this work will take.